support-frontend/build.sbt

```diff
@@ -43,6 +43,10 @@ libraryDependencies ++= Seq(
 )
 dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-databind" % jacksonDatabindVersion
 
+ThisBuild / libraryDependencySchemes ++= Seq(
+  "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
+)
```

Comment:
this was caused by conflicting versions of scala-xml which SBT worries will cause a binary incompatiblity, so it aborts at that point.  In reality it doesn't cause an issue, this also happens elsewhere, and the usual suggestion is to add this line.  E.G. https://github.com/sbt/sbt/issues/6997#issuecomment-1310637232